instruments/tests/test_named_struct.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Module containing tests for named structures.
+"""
+
+# IMPORTS ####################################################################
+
+from __future__ import absolute_import
+
+from instruments.named_struct import (
+    Field, StringField, Padding, NamedStruct
+)
+
+# TESTS ######################################################################
+
+# pylint: disable=no-member,protected-access,blacklisted-name,missing-docstring
+
+def test_named_struct_roundtrip():
```

Comment:
These might be a good candidate to use [Hypothesis](https://github.com/HypothesisWorks/hypothesis-python) on, where we can try a wide variety of input values. I'll give it a try and see what we get.